docs/guide/tournament-logistics.rst

```diff
@@ -463,7 +463,7 @@ Mistakes are made surprisingly often during results announcements. Again, this i
 - Have a single source for what is being read out — i.e. a printed list (recommended) or the tab site itself — but don't mix and match. If making slides (often a good idea for large/crowded venues) copy the data from the canonical source being announced.
 - Double check what is being read out against the tab site, and/or whatever draft lists were used to determine the adjudicator's break. Verify with the adjudication core that everyone who should be there is, and that nobody is missing.
 - Clarify what information should be on the print-outs and the general order in which things are read. For example, it might be easy to omit breaking adjudicator's institutions, to use ambiguous abbreviations over full institution names, or to have an inconsistent approach to how the information is read (i.e. whether it is read as *wins* then *team points* then *team name*).
-- Without revealing any details try and get at least some guidance on how to pronounce names that people are not familiar with pronounce.
+- Without revealing any details try and get at least some guidance on how to pronounce names that people are not familiar with.
```

Comment:
How about "Without revealing any details, try to get at least some guidance on how to pronounce unfamiliar names."?

Reply:
Rephrased to "`- Without revealing any details, try to get some guidance on how to pronounce unfamiliar names.`"